fossid-webapp-client/src/main/kotlin/api/Project.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Project {
+    var id: Int? = null
+
+    var created: String? = null
+    var updated: String? = null
+    var creator: String? = null
+
+    @JsonProperty("project_code")
```

Comment:
BTW, all these annotations would not be necessary when using a mapper with snake-style property naming strategy, [like we do for ORT](https://github.com/oss-review-toolkit/ort/blob/ce590f5d6d7151c89c852d621a53e51e416bf0b9/model/src/main/kotlin/Mappers.kt#L44). I acknowledge we must not depend on ORT code here, but I wonder whether we should instead provide a convenience function to create the proper mapper / mapper configuration, and in exchange get rid of all these annotations (which are a pain to maintain).